### PR TITLE
feat: trade no longer links to opposite trade

### PR DESCRIPTION
**Pull request by pawel-betdex**
dropping opposite trade from trade account in anticipation of cross matching (when more than two orders constitute a match)

### CODE REVIEW

File: programs/monaco_protocol/src/state/trade_account.rs
Comment (eoin-betdex) [inline programs/monaco_protocol/src/state/trade_account.rs:10]: Account size can be reduced too

**Review by eoin-betdex — COMMENTED** [review]
Will take a second look soon

File: programs/monaco_protocol/src/state/trade_account.rs
Comment (pawel-betdex) [reply programs/monaco_protocol/src/state/trade_account.rs:10]: good point

**Review by eoin-betdex — APPROVED** [review]
LGTM 👍 

Please wait until #181 is merged and `release/0.14.0` is created before merging.

**Review by obi-ewan — APPROVED** [review]

**Pull request merged**
